chef/cookbooks/glance/templates/default/glance-cache.conf.erb

```diff
@@ -31,6 +31,9 @@ registry_host = <%= @registry_bind_host %>
 # Port the registry server is listening on
 registry_port = <%= @registry_bind_port %>
 
+<% # Missing in upstream template -%>
```

Comment:
No upstream bug, fixed in Mitaka (iirc): glance config files are not automatically generated upstream.